code/_globalvars/lists/objects.dm

```diff
@@ -32,6 +32,7 @@ GLOBAL_LIST_EMPTY(meteor_list)				// List of all meteors.
 GLOBAL_LIST_EMPTY(active_jammers)             // List of active radio jammers
 GLOBAL_LIST_EMPTY(ladders)
 GLOBAL_LIST_EMPTY(trophy_cases)
+GLOBAL_LIST_EMPTY(echo_images)

```

Comment:
is there a reason to have this as a global list and not just a list on SSoutputs?